pageserver/src/walrecord.rs

```diff
@@ -3,12 +3,11 @@
 //!
```

Comment:
I've got a feeling that we will have more version-specific differences in this file in the future. This is ok for now, but we'll probably need to refactor it more in the future.